Gaijinizer patch/Map217.txt

```diff
@@ -776,7 +776,7 @@ I'd lived my life.#Ron
 条件【\\C[2]友好度９０以上\\C[0]】
 > CONTEXT: Map217/events/5/pages/1/243/ChoiceHelp
 > CONTEXT: Map217/events/5/pages/2/243/ChoiceHelp
-conditions【\\C[2] Degree of friendship 90 or more\\C[0] ]#MTLed
+Requires: 【 \\C[2]Friendship 90+\\C[0] ]#Strate
 > END STRING
```

Comment:
Change the "special" left bracket to a regular one.

Reply:
done